docs/guide/plotting.rst

```diff
@@ -88,30 +72,23 @@ and access the figures and axes objects. This is shown in the following example.
     ax.set_xlabel('x')
     plt.show()
 
-In matplotlib, `~matplotlib.figure.Figure` is the top-level container for all plot elements and
-`~matplotlib.axes.Axes` is the top-level container for a particular plot. So the above example,
-creates a figure then creates an axes and populates the plot in ``ax``. With this method you
-now have your hands on the `~matplotlib.axes.Axes` object so you can do things
+In Matplotlib, `~matplotlib.figure.Figure` is the top-level container for a single figure and `~matplotlib.axes.Axes` is the top-level container for a set of axes.
+The above example creates a figure then creates an axes and populates the plot in ``ax``.
+With this method you now have your hands on the `~matplotlib.axes.Axes` object so you can do things
 like change the labels on the x and y axes or add a legend.
-In the previous section, pyplot took care of creating these
-objects for you so you didn't have to worry about creating them yourself.
-
-4. sunpy Plotting Convention
-****************************
+In the previous section, pyplot took care of creating these objects for you so you didn't have to worry about creating them yourself.
 
-To be consistent with matplotlib, sunpy has developed a standard plotting policy
-which supports both simple and advanced matplotlib usage. The following examples
-focus on the map object but they should be applicable across all of the data
-objects.
+Plotting in sunpy
+*****************
 
-4.1 peek()
-**********
+To be consistent with Matplotlib, sunpy has developed a standard plotting interface which supports both simple and advanced Matplotlib usage.
+The following examples focus on the map object, but both ``peek()`` ``plot()`` work on time series objects too.
 
-For quick and easy access to a plot all sunpy base objects (i.e. maps, spectra,
-timeseries) define their own `~sunpy.map.mapbase.GenericMap.peek` command which
-will create a plot for you and show it without you having to deal with any
-matplotlib setup. This is so that it is easy to take a quick look at your data.
-For example you can make the following plot.
+peek()
```

Comment:
I wonder if we want to link these to the GenericMap methods?

Reply:
🤷